_posts/2016-01-04-researcher-survey.md

```diff
@@ -0,0 +1,16 @@
+---
+name: 2016-user-survey
+layout: post
+title: Help us prioritize what to build in 2016
+date: 2016-01-04
+authors:
+  - name: Karthik Ram
+tags:
+- survey
+---
+
+We've got a big year ahead of us as we work towards expanding our team and organizing various events and activities. We remain committed to supporting and expanding the landscape of open source tools that are available to researchers. While much of our focus has been around making it easier to access various data repositories, we are keen on improving other parts of the research pipeline, including data munging, documentation and sharing. To help us prioritize our development activities and better understand the needs of researchers, we would really appreciate 5-12 minutes of your time to complete a survey:
```

Comment:
data? Should I rephrase that sentence? (the munging, documentation and sharing all refer to data)